brand-guidelines/color-principles.md

```diff
@@ -24,945 +24,109 @@ A minimal palette, clear hierarchy, good information design, and ample whitespac
 
```

Comment:
This is as low as I can place this comment, but please update the page intro text above here to reflect the latest in the working doc.